test/testJavaScriptFileType.js

```diff
@@ -1,7 +1,7 @@
 /*
  * testJavaScriptFileType.js - test the HTML template file type handler object.
  *
- * Copyright © 2019, Box, Inc.
+ * Copyright © 2019, JEDLSoft
```

Comment:
Should probably be LGE, Inc. no?

Reply:
Same opinion with `ilib-loctool-webos-json-resource` plug-in.
For now, I think it's ok to JEDLSoft as opensource work as POC step. then when it comes to adopting actual webOS products, it's better to update LGE Inc.